man/systemd.network.xml

```diff
@@ -2257,6 +2257,53 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
```

Comment:
Just a suggestion from a passerby... `TrafficControlQueueingDiscipline` seems very long to me, is there any way to make it shorter somewhat? Like, drop `Queueing` maybe? or leave just `QueueingDiscipline`?

Reply:
No this is perfect. It makes clearer.